packages/ui-components/src/components/Message/Message.component.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState, useRef } from "react"
+
+import { Icon } from "../Icon"
+
+import { MessageProps } from "./Message.types"
+import { getMuiIcon } from "./utils/iconUtils"
+import { initiateAutoDismiss, clearAutoDismissTimeout } from "./utils/timeoutUtils"
```

Comment:
I would prefer to keep this inline in the `Message` component right now. Mostly to be in line with other similar components where it is like this, too. It makes it harder to see what the functions do because you have to jump to another file. 

In general I'm not against this type of refactoring but then we need to look at it holistically and do the same thing in all components where it applies to ensure consistency.

Reply:
Thank You for the review, Esther. No problem, I can do that. For me, it makes the component code very hard to read. I like to open a component file and see quickly what it does. Other helpers, especially css variables etc. are secondary to that, in my opinion. 

Overall it was a really nice way for me to play around with the code and get familiar with it. 🤪

But the most important thing indeed is consistency. If the team prefers to structure code this way across all components and find it easier to understand the full logic in one place, I agree we should keep it this way. Thank You. 🚀